Scripts/Runtime/VoiceService.cs

```diff
@@ -70,7 +84,14 @@ public interface IVoiceService
         /// </summary>
         void Activate();
 
+        /// <summary>
+        /// Activate the microphone and send data for NLU processing with custom request options.
+        /// </summary>
+        /// <param name="requestOptions"></param>
+        void Activate(WitRequestOptions requestOptions);
```

Comment:
can we provide this as default value for the parameter and dedup the methods?

Reply:
Default values need to be compile-time constants. The default parameter such as `WitRequestOptions requestOptions = new WitRequestOptions()` isn't a compile-time constant, so was running into issues there. @yolanother do you know a better way to get around this?